src/diagonal.works/b6/cmd/b6/js/static/b6.css

```diff
@@ -604,12 +605,12 @@ form.shell-form > input {
     flex: 0 1 auto;
     padding-right: 5px;
     padding-left: 5px;
-    color: #528DFF;
+    color: #528dff;
```

Comment:
I usually use [Prettier](https://prettier.io/) for formatting, it can be configured on VSCode to format on save as well. I'll open a different PR to add a prettier config file for the JS part of the project :)